assets/scss/styles.scss

```diff
@@ -208,6 +193,10 @@ h1 {
             border-bottom: 2px solid #0b0c0c;
           }
         }
+        &.row_centre_occupied,
+        &.row_centre_capacity {
+          border-bottom: 1px solid #BFC1C3;
```

Comment:
As before, this is also defined in the gov.uk palette, in this instance as $grey-X (replace X with a number for different shades of grey)